selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/miscellaneous/WorkingWithTerminalTest.java

```diff
@@ -92,7 +92,8 @@ private void prepareNewTerminal() {
         terminal.waitTerminalIsNotPresent(1);
       }
 
-      consoles.openNewTerminalIntoProcesses();
+      consoles.clickOnPlusMenuButton();
+      consoles.clickOnTerminalItemInContextMenu();
```

Comment:
let's divide *consoles* and *terminal* groups of commands by empty line to improve readability of code